code/code/misc/spell_info.cc

```diff
@@ -3421,7 +3421,7 @@ void buildSpellArray() {
       "original form.",
       "$n's limbs shimmer and momentarily resemble their original form.",
       START_21, LEARN_4, START_DO_20, LEARN_DO_5, START_DO_NO, LEARN_DO_NO,
-      LEARN_DIFF_SPELLS, 0.04, COMP_MATERIAL, 0);
+      LEARN_DIFF_SPELLS, 0.04, COMP_GESTURAL | COMP_GESTURAL_RANDOM | COMP_VERBAL | COMP_VERBAL_RANDOM | COMP_MATERIAL | COMP_MATERIAL_END | SPELL_TASKED, 0);
```

Comment:
Make sure you run clang-format on this file too, just to keep formatting consistent.